charts/px-central/templates/px-lighthouse/px-central-oidc/pxcentral-keycloak.yaml

```diff
@@ -462,6 +462,8 @@ spec:
               value: /secrets/password
             - name: JAVA_TOOL_OPTIONS
               value: "-XX:+UseContainerSupport -XX:MaxRAMPercentage=50.0"
+            - name: JAVA_OPTS
+              value: "-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true"
```

Comment:
Is this related to IPV6 change ?

Reply:
keycloak figures out the IPV6 but only if the the JAVA_OPTS is provided then it actually does it but otherwise it will go with default flow. So this is a way to force keycloak to figureout the IPV4 or V6 internally.

Reply:
Thanks for the info!